rage/rlWorker.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+namespace rage
+{
+    class rlWorker
```

Comment:
so, for what is that?

Reply:
> so, for what is that?

It is used for rewards for achievement, progress in missions, etc